modules/bibdk_heimdal/includes/bibdk_heimdal.cron.inc

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Why is the cron in a separate file? I think it should be in the .module so php doesn't have to open yet another file...

Their isn't that much content here.